.byebug_history

```diff
@@ -0,0 +1,33 @@
+exit
```

Comment:
Add this file to gitignore.

Remember to refresh the git cache so this file disappears from the PR.